src/classes/sendEmail.ts

```diff
@@ -31,12 +31,18 @@ import { SendMailOptions, SendEmailResult } from '../types'
  * })
  * ```
  *
- * #### Options
+ * ## Options
  *
  * This function uses the popular
  * [`nodemailer`](https://www.npmjs.com/package/nodemailer) internally to send
  * emails. Please see the documentation for [Message
  * Configuration](https://nodemailer.com/message/) for available options.
+ *
+ * @returns
+ *
+ *   ## Role Permissions Required
+ *
+ *   No Permissions Required
```

Comment:
This function requires AWS credentials. I dont think we need to mention roles here, better off mentioning the IAM policy required.